src/metric-engine/src/metadata_region.rs

```diff
@@ -122,26 +122,9 @@ impl MetadataRegion {
     /// This method doesn't check if the previous key exists.
     pub async fn remove_logical_region(
         &self,
-        physical_region_id: RegionId,
+        _physical_region_id: RegionId,
         logical_region_id: RegionId,
     ) -> Result<()> {
-        // concat region key
-        let region_id = utils::to_metadata_region_id(physical_region_id);
-        let region_key = Self::concat_region_key(logical_region_id);
-
-        // concat column keys
-        let logical_columns = self
-            .logical_columns(physical_region_id, logical_region_id)
-            .await?;
-        let mut column_keys = logical_columns
-            .into_iter()
-            .map(|(col, _)| Self::concat_column_key(logical_region_id, &col))
-            .collect::<Vec<_>>();
-
-        // remove region key and column keys
-        column_keys.push(region_key);
-        self.delete(region_id, &column_keys).await?;
-
```

Comment:
What will happen if we just want to drop a logical region?

Reply:
seems we can not here i searched the code base this function not affect that

Reply:
This part may need to be retained when dropping a logic table, but there should also be a fast path to execute `drop phy_table`. cc @WenyXu @waynexia

Reply:
if add a fast_path arg in Drop database seems need to through some functions 
do we need that? maybe I can fix it in this way after some discuss.

Reply:
> if add a fast_path arg in Drop database seems need to through some functions do we need that? maybe I can fix it in this way after some discuss.

We can add a fast_path in the drop database procedure. If the datanode receives a drop physical table request with a force flag, it will drop all logical tables and the physical table in a single request.

Reply:
> > if add a fast_path arg in Drop database seems need to through some functions do we need that? maybe I can fix it in this way after some discuss.
> 
> We can add a fast_path in the drop database procedure. If the datanode receives a drop physical table request with a force flag, it will drop all logical tables and the physical table in a single request.

yes that's my point, it will go through some functions if that is OK, I will do it in This way

Reply:
find it is kind of hard add it in the drop database procedure